### ISSUE TITLE
Abbreviated forms not recognized

### ISSUE THREAD

**Issue by junkblocker** [haya14busa/go-vimlparser#19]
Short forms like ``endfu`` for ``endfunction`` etc. are not recognized.

**Comment by haya14busa** [comment]
It works.

```
$ echo 'func F() | endf' | vimlparser
(function (F))
```

**Comment by junkblocker** [comment]
Here's an example where it doesn't for ``:diffthis``.

```
echo 'difft' | vimlparser
vimlparser: E492: Not an editor command: difft: line 1 col 1
```

**Comment by haya14busa** [comment]
Thanks I fixed it (#20).

I also opened p-r to original vim-vimlparser repo. https://github.com/ynkdir/vim-vimlparser/pull/31

**Comment by junkblocker** [comment]
Thanks, @haya14busa !
